src/deluge/gui/views/automation_instrument_clip_view.cpp

```diff
@@ -1154,33 +1328,24 @@ ActionResult AutomationInstrumentClipView::buttonAction(hid::Button b, bool on,
 
 	else {
 passToOthers:
-
 		uiNeedsRendering(this);
 
+		if (on && (b == PLAY) && display->have7SEG() && playbackHandler.isEitherClockActive()
+		    && !isOnAutomationOverview() && !padSelectionOn) {
+			playbackStopped = true;
+		}
+
 		ActionResult result = InstrumentClipMinder::buttonAction(b, on, inCardRoutine);
 		if (result != ActionResult::NOT_DEALT_WITH) {
 			return result;
 		}
 
 		result = ClipView::buttonAction(b, on, inCardRoutine);
 
```

Comment:
no need these cancelPopup calls as we're no longer using popups in the automation view to display parameter name or parameter value